lib/seattleflu/id3c/api/routes.py

```diff
@@ -109,11 +109,11 @@ def get_scan_demographics(session):
 @authenticated_datastore_session_required
 def get_scan_positives(session):
```

Comment:
Should this function be renamed?

Reply:
Gernot had asked to add it to the existing endpoint, so I assume they don't want to change the API call on their end.